packages/inter-protocol/test/price/test-fluxAggregator.js

```diff
@@ -55,9 +55,10 @@ const makeContext = async () => {
     );
 
     const aggregator = await makeFluxAggregator(
+      baggage,
       zcfTestKit.zcf,
       manualTimer,
-      { ...quoteIssuerKit, assetKind: 'set', displayInfo: undefined },
+      { ...quoteIssuerKit, displayInfo: { assetKind: 'set' } },
```

Comment:
That can't be right, can it?  `assetKind` isn't part of `displayInfo`. What's going on?

Reply:
this is actually an [AdditionalDisplayInfo](https://github.com/Agoric/agoric-sdk/blob/6d3db5a7bcb2fc636de9f23a8c63266a981898a1/packages/ERTP/src/types-ambient.js#L263-L278)